src/main/java-gen/io/dronefleet/mavlink/ardupilotmega/LogStartNew.java

```diff
@@ -0,0 +1,126 @@
+package io.dronefleet.mavlink.ardupilotmega;
+
```

Comment:
@tomerg-flytrex 
Is this code generated automatically?

Reply:
Yes, the only things I changed are the xml and the build.gradle